discourse-setup

```diff
@@ -316,6 +325,17 @@ read_default() {
   read_default_result=`echo $read_config_result | sed "s/^\([\"']\)\(.*\)\1\$/\2/g"`
 }
 
+assert_maxmind_license_key() {
+  echo "Assert maxmind license key for $web_file"
+  if  ! grep DISCOURSE_MAXMIND_LICENSE_KEY $web_file
+  then
+    echo "Adding DISCOURSE_MAXMIND_LICENSE_KEY to $web_file"
+    sed -i -e 's/LETSENCRYPT_ACCOUNT_EMAIL/a\ \ #DISCOURSE_MAXMIND_LICENSE_KEY: 1234567890123456' $web_file 
+  else
+    echo "Have maxmind already"
```

Comment:
Since this is adding the commented-out version, this log line is misleading and should be removed. How about making the previous one "Adding .... placeholder to $web_file" ?

Reply:
How about this:

Reply:
lgtm